tests/components/Pagination.test.tsx

```diff
@@ -0,0 +1,168 @@
+import { render, screen, within } from '@testing-library/react';
+import { State, VerticalSearchState } from '@yext/answers-headless-react';
+import { spyOnActions, mockAnswersState, mockAnswersHooks } from '../__utils__/mocks';
+import userEvent from '@testing-library/user-event';
+import { Pagination } from '../../src/components/Pagination';
+import { verticalNoResults } from '../__fixtures__/data/noresults';
+
+const mockedState: Partial<State> = {
+  searchStatus: {
+    isLoading: false
+  },
+  meta: {
+    searchType: 'vertical'
+  },
+  query: {}
+};
+
+const mockedActions = {
+  state: mockedState,
+  setOffset: jest.fn(),
+  executeVerticalQuery: jest.fn()
+};
+
+jest.mock('@yext/answers-headless-react');
+
+beforeEach(() => {
+  mockAnswersHooks({ mockedState, mockedActions });
+});
+
+describe('no results are returned from search', () => {
+  it('doesn\'t display pagination when paginateAllOnNoResults is false', () => {
```

Comment:
I view this test as more of there's nothing to display (no results from the search and from alternativeresults section in the mocked state) so there's no pagination component, not necessarily because paginateAllOnNoResults is false.

Reply:
renamed the test and rendered the pagination component without props (defaults to paginateAllOnNoResults=false)